docs/script.rst

```diff
@@ -371,7 +367,7 @@ Here's how it works::
 
 
 
-Applying some syntax sugar, you can do the same in just one line of code::
+Applying some syntactic sugar, you can do the same in just one line of code::
```

Comment:
"Syntactic sugar" is the term in common use (if you Google "syntax sugar", you get "syntactic sugar").

Reply:
👍